src/main/java/figures/Circle.java

```diff
@@ -0,0 +1,22 @@
+package figures;
+
+public class Circle extends Figure {
+
+    public double radius = Math.pow(super.getSquare(), 0.5);
+
+    public Circle() {
+    }
+
+    public double getRadius() {
+        return this.radius;
+    }
+
+    @Override
+    public String toString() {
+        return "Circle{" +
+                "radius=" + radius +
```

Comment:
```suggestion
                " radius= " + radius +
```
don't forget about spaces